docs/architecture-decisions/0004-monolithic-application.md

```diff
@@ -0,0 +1,53 @@
+# 4. Monolithic application
```

Comment:
👍🏻 on this ADR

Reply:
👍 On this ADR. (FYI, I pushed a few commits fixing typos; none of them changed any wording.)

Reply:
👍 on this ADR